mongo_orchestration/servers.py

```diff
@@ -418,6 +425,9 @@ def shutdown(self):
             return
         logger.info("Attempting to connect to %s", self.hostname)
         client = self.connection
+        # If we have set requireApiVersion, disable it.
+        if self.require_api_version:
+            client.admin.command("setParameter", 1, requireApiVersion=0)
```

Comment:
Is disabling requireApiVersion required before a shutdown?

Reply:
I added it based on Jesse's comment: https://jira.mongodb.org/browse/PYTHON-2520?focusedCommentId=3583776&focusedId=3583776&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-3583776